files/architecture/backend_architecture.tex

```diff
@@ -0,0 +1,15 @@
+\section{Back-end architecture}
+The back-end application is written in TypeScript and build with a framework called Apollo.
+The back-end is spun up using Express and \gls{node}
+
+The back-end contains 3 layers of abstraction.
+
+\graphic{1}{backend_architecture}{Back-end layer}{fig:backendLayer}
+
+The top layer is the one that gets exposed to the users of the back-end. The Schema layer is all about defining the schema that is exposed. The schema exposes models and functions that the user can query for. The Schema is written in the GraphQL Schema Language.
+
+When the user queries, Apollo looks for a resolve function to resolve the query. This layer is called the Resolvers layer. In this layer, functions are defined to resolve specific queries related to the schema. 
```

Comment:
To make the sentences more readable, consider merging them into longer once like the last two:
From:
`This layer is called the Resolvers layer. In this layer, functions are defined to resolve specific queries related to the schema. `
To:
`This layer is called the Resolvers layer. Here, functions are defined to resolve specific queries related to the schema. `